_includes/swc/setup.html

```diff
@@ -1,538 +1,27 @@
-              {% comment %} Installing {% endcomment %}
-              {% comment %} Completing the Git Setup Wizard {% endcomment %}
-              <li>Click on "Finish".</li>
-            </ol>
-          </li>
-          <li>
-            If your "HOME" environment variable is not set (or you don't know what this is):
-            <ol>
-              <li>Open command prompt (Open Start Menu then type <code>cmd</code> and press [Enter])</li>
-              <li>
-                Type the following line into the command prompt window exactly as shown:
-                <p><code>setx HOME "%USERPROFILE%"</code></p>
-              </li>
-              <li>Press [Enter], you should see <code>SUCCESS: Specified value was saved.</code></li>
-              <li>Quit command prompt by typing <code>exit</code> then pressing [Enter]</li>
-            </ol>
-	  </li>
-        </ol>
-        <p>This will provide you with both Git and Bash in the Git Bash program.</p>
-      </article>
-      <article role="tabpanel" class="tab-pane active" id="shell-macos">
-        <p>
-          The default shell in some versions of macOS is Bash, and
-	  Bash is available in all versions, so no need to install anything.
-	  You access Bash from the Terminal (found in
-	  <code>/Applications/Utilities</code>).
-          See the Git installation <a href="https://www.youtube.com/watch?v=9LQhwETCdwY ">video tutorial</a>
-          for an example on how to open the Terminal.
-          You may want to keep Terminal in your dock for this workshop.
-        </p>
-        <p>
-             To see if your default shell is Bash
-            type <code>echo $SHELL</code> in Terminal and press the enter/return key.
-            If the message printed does not end with '/bash' then your default is something else and you can run
-            Bash by typing <code>bash</code>
-        </p>
-        <p>
-          If you want to change your default shell, see <a href="https://support.apple.com/en-au/HT208050" rel="noopener">
-          this Apple Support article</a> and follow the instructions on "How to change your default shell".
-        </p>
-      </article>
-      <article role="tabpanel" class="tab-pane active" id="shell-linux">
-        <p>
-          The default shell is usually Bash and there is usually no need to
-          install anything.
-        </p>
-        <p>
-          To see if your default shell is Bash type <code>echo $SHELL</code> in
-          a terminal and press the enter/return key.  If the message printed does
-	  not end with '/bash' then your default is something else and you can run
-	  Bash by typing <code>bash</code>.
-        </p>
-      </article>
-    </div>
-  </div>
-</div> {% comment %} End of 'shell' section. {% endcomment %}
+{% if site.flavor == "r" %}
+{% include install_instructions/python.html %}
+{% elsif site.flavor == "python" %}
+{% include install_instructions/r.html %}
+{% else %}
+{% include warning-flavor.html %}
+{% endif %}
```

Comment:
Should these install instructions be flipped?

```suggestion
{% if site.flavor == "r" %}
{% include install_instructions/r.html %}
{% elsif site.flavor == "python" %}
{% include install_instructions/python.html %}
{% else %}
{% include warning-flavor.html %}
{% endif %}
```